dolphinscheduler-ui/src/views/resource/file/upload/index.tsx

```diff
@@ -55,6 +53,8 @@ export default defineComponent({
 
     const removeFile = () => {
       state.uploadForm.name = ''
+      state.uploadForm.file = ''
+      state.uploadFormRef.validate()
```

Comment:
No verification is required here.

Reply:
Agree. I'll add a commit to cancel the verification.